### PR TITLE
Added Middle East Bahrain region

### PR DESCRIPTION
**Pull request by juliancheal**
Fixes BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1737459

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -35,7 +35,12 @@
 
     # disable regions by their keys found in app/models/manageiq/providers/amazon/regions.rb e.g.
     # - us-gov-west-1
-    :disabled_regions: []
+    :disabled_regions:
```
Comment (juliancheal) [inline config/settings.yml]: @agrare I ran `rake aws:extract:regions` was I supposed to? It's added all the other disabled regions?

**Comment by juliancheal** [comment]
@miq-bot assign @agrare

File: config/settings.yml (hunk 1 above)
Comment (agrare) [reply config/settings.yml]: Hey @juliancheal we don't disable any regions upstream as of 868311b6c5761f1c9f237ecfbba992cd748e0785
Did that rake task add these to the disabled_regions?

Comment (juliancheal) [reply config/settings.yml]: Ah cool, thanks. Yes it did. Ok I'll remove those lines in this PR. And take a look at the rake task.

Comment (agrare) [reply config/settings.yml]: Awesome, yeah Alex must have forgotten to update the rake task.

**Comment by simaishi** [comment]
@juliancheal can this be `ivanchuk/yes`?

**Comment by juliancheal** [comment]
@simaishi yes.

**Comment by juliancheal** [comment]
@miq-bot add_label ivanchuk/yes

**Comment by simaishi** [comment]
Ivanchuk backport details:

```
$ git log -1
commit 962f313403ef4af1eb5f295719a777bcad6f0b6a
Author: Adam Grare <agrare@redhat.com>
Date:   Fri Aug 23 10:20:46 2019 -0400

    Merge pull request #552 from juliancheal/add_bahrain_region
    
    Added Middle East Bahrain region
    
    (cherry picked from commit e84a5c526b3b69791c5c3fc15e9b23d4fa599ae2)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1767775
```

**Pull request merged**